ucsmsdk_samples/firmware/ucsfirmware.py

```diff
@@ -52,6 +52,20 @@ def firmware_available(username, password, mdf_id_list=None, proxy=None):
     image_names = [image.image_name for image in images]
     return sorted(image_names)
 
+def return_download_state(handle, file_name):
```

Comment:
Don't think we need this function just for querying `state`.
Can be a part of the `firmware_add_remote` function itself.